src/main/java/com/minecolonies/coremod/colony/Structures.java

```diff
@@ -153,6 +148,50 @@ private static void loadStyleMaps()
         }
     }
 
+
+    private static void loadStyleMapsJar()
+    {
+        try
+        {
+            Log.getLogger().info("loadStyleMaps jar: " + ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI()); 
+            final URI uri = ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI();
+            if ("jar".equals(uri.getScheme()))
+            {
+                try (FileSystem fileSystem = FileSystems.getFileSystem(ColonyManager.class.getResource(SCHEMATICS_ASSET_PATH).toURI()))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1141)